src/main/java/com/ericsson/ei/controller/FailedNotificationController.java

```diff
@@ -10,22 +10,22 @@
 
 
 /**
- * This class searches for missed notifications in the database.
+ * This class searches for failed notifications in the database.
  * (Generated with springmvc-raml-parser v.2.0.4)
  * 
  */
 @RestController
 @Validated
-@RequestMapping(value = "/queryMissedNotifications", produces = "application/json")
-public interface QueryMissedNotificationController {
+@RequestMapping(value = "/failedNotifications", produces = "application/json")
+public interface FailedNotificationController {
```

Comment:
The class name should be plural: FailedNotificationsController

Reply:
done

Reply:
Since raml generated a new class I'm guessing this can be removed?

Reply:
I think raml generates it as singular

Reply:
I noticed RAML overrides the naming in the controller classes, is it possible to configure it to be in plural or are we stuck with using the singular form of "FailedNotificationController"?

Reply:
It seems that RAML truncate the "s" and we can't make it plural through RAML. 

Nevertheless, removing "s" is grammatically correct.

Reply:
Ok, good to know that RAML forces the use of certain naming in the controllers and their methods ...